website/docs/addon_unreal_artist.md

```diff
@@ -667,20 +667,21 @@ Once the render is finished, you can publish the render. Click on the AYON icon
 
 ![Unreal AYON Tools Publish](assets/unreal_ayon_menu_publisher.png)
 
-### Farm rendering
+### Farm Rendering
 
-Unreal integration also supports rendering on Deadline. Process of creation of render instance is same as higher, difference is in
-selection of `Farm rendering` in `Render target`.
+Unreal integration also supports rendering on Deadline. The process for creating a render instance is the same as mentioned above, but you need to select `Farm Rendering` in `Render Target`.
 
-Deadline rendering process expects physically existing Render Queue and Render Settings uassets in the Unreal project.
+AYON requires `Render Queue` and `Render Configuration` uassets to be physically present in the Unreal project.
```

Comment:
Actually these config files are mandatory also when local rendering happens...I guess we need to add it also at the begining of "Render Section" in the user docs or just  move it there (single mentioning for both local/farm rendering)...